<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择待添加物料</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>581</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: rgba(255, 255, 255, 0);
    border: 2px solid rgb(84, 110, 122);
    border-radius: 5px;
    margin-top: 1em;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="checkBox_mn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Mn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_sb">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>255</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Sb</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_cusi">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Cu-Si</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_cup">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Cu-P</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ni">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Ni</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_cuzr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>75</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Cu-Zr</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_in">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>In</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_zn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>165</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Zn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ag">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Ag</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_sn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>165</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Sn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_cu">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>75</y>
      <width>180</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 50px; 
}

QCheckBox::indicator {
    width: 36px;
    height: 36px;
}

 
</string>
    </property>
    <property name="text">
     <string>Cu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
